benches/benches/db_lookup_times.rs

```diff
@@ -0,0 +1,97 @@
+use crate::db_lookup_times_utils::{
+    matrix::matrix,
+    utils::{
+        get_full_block,
+        get_random_block_height,
+        multi_get_block,
+        open_db,
+        open_raw_rocksdb,
+        thread_rng,
+    },
+};
+use criterion::{
+    criterion_group,
+    criterion_main,
+    Criterion,
+};
+use db_lookup_times_utils::seed::{
+    seed_compressed_blocks_and_transactions_matrix,
+    seed_full_block_matrix,
+};
+use fuel_core_storage::transactional::AtomicView;
+
+mod db_lookup_times_utils;
+
+pub fn header_and_tx_lookup(c: &mut Criterion) {
+    let method = "header_and_tx";
+    let mut rng = thread_rng();
+
+    seed_compressed_blocks_and_transactions_matrix(method);
+    let mut group = c.benchmark_group(method);
+
+    for (block_count, tx_count) in matrix() {
+        let database = open_db(block_count, tx_count, method);
+        let view = database.latest_view().unwrap();
+        group.bench_function(format!("{block_count}/{tx_count}"), |b| {
+            b.iter(|| {
+                let height = get_random_block_height(&mut rng, block_count);
+                let block = view.get_full_block(&height);
+                assert!(block.is_ok());
+                assert!(block.unwrap().is_some());
+            });
+        });
+    }
+
+    group.finish();
+}
+
+pub fn multi_get_lookup(c: &mut Criterion) {
+    let method = "multi_get";
+    let mut rng = thread_rng();
+
+    seed_compressed_blocks_and_transactions_matrix(method);
+    let mut group = c.benchmark_group(method);
+
+    for (block_count, tx_count) in matrix() {
+        let database = open_raw_rocksdb(block_count, tx_count, method);
+        group.bench_function(format!("{block_count}/{tx_count}"), |b| {
+            b.iter(|| {
+                let height = get_random_block_height(&mut rng, block_count);
+                // todo: clean up
```

Comment:
It's not obvious to me which code this comment refers to. Is this a stale todo that was addressed by factoring out the `multi_get_block` function or are there any other cleanups you intend?

Reply:
i want to assert here that the multi_get_block lookup succeeded :)